dags/pegs-challenge-submission-dag.py

```diff
@@ -10,17 +10,17 @@
 
 dag_params = {
     "synapse_conn_id": Param("SYNAPSE_ORCA_SERVICE_ACCOUNT_CONN", type="string"),
-    "synapse_evaluation_id": Param("9615531", type="string"),
+    "synapse_evaluation_id": Param("9615511", type="string"),
     "tower_conn_id": Param("PEGS_CHALLENGE_PROJECT_TOWER_CONN", type="string"),
     "tower_run_name": Param("pegs_model_submission_evaluation", type="string"),
-    "tower_view_id": Param("syn53239158", type="string"),
+    "tower_view_id": Param("syn55253884", type="string"),
     "tower_input_id": Param("syn53239289", type="string"),
     "tower_compute_env_type": Param("spot", type="string"),
 }
 
 dag_config = {
-    "schedule_interval": None,
-    "start_date": datetime(2023, 6, 1),
+    "schedule_interval": "*/1 * * * *",
```

Comment:
I used `* * * * *` for my "run every minute" chron expression. I'm assuming this does the same thing, but just wanted to note it.

Reply:
@BWMac This is a site I reference often: https://crontab.guru/#*/1_*_*_*_*

Reply:
@BryanFauble Thanks for the link, that's a good reference to have in the back pocket. And yup it confirmed that both `*/1 * * * *` and `* * * * *` mean the same thing.